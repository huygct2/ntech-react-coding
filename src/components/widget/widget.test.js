import React from 'react';
import renderer from 'react-test-renderer';
import moment from 'moment';
import Widget from './widget';

test('Widget component', () => {
  const time = moment();
  const component = renderer.create(
    <Widget
      typeBet="Soccer"
      time={time}
      timeNow={time}
      title="EPL"
      description="Arsenal"
    />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
