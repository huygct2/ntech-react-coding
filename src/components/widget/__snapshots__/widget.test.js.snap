// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Widget component 1`] = `
<div
  className="card widget"
>
  <div
    className="card-image"
  >
    <div
      className="filter-black"
    />
    <figure
      className="image is-2by1"
    >
      <img
        alt="background"
        src=""
      />
    </figure>
    <div
      className="box-left has-text-weight-bold is-uppercase"
    >
      <img
        alt="logo sport"
        src="logo-ball.png"
      />
      <span>
        Soccer
      </span>
    </div>
    <div
      className="box-right has-text-white has-text-weight-bold"
    >
      <time>
        Starts in 
        <span>
          <span
            className="amount-label"
          >
            0
          </span>
          <span
            className="unit-label"
          >
            s
             
          </span>
        </span>
      </time>
    </div>
    <div
      className="content has-text-white"
    >
      <h2
        className="content__title has-text-weight-bold is-uppercase"
      >
        EPL
      </h2>
      <span
        className="content__description has-text-weight-semibold"
      >
        Arsenal
      </span>
    </div>
  </div>
  <div
    className="card-content"
  >
    <div />
  </div>
</div>
`;
